packages/react/src/primitives/Tabs/__tests__/Tabs.test.tsx

```diff
@@ -42,6 +42,32 @@ describe('Tabs: ', () => {
     expect(ref.current.nodeName).toBe('DIV');
   });
 
+  it('should skip over null children', async () => {
+    render(
+      <Tabs testId="tabsTest">
+        <TabItem title="Tab 1">Tab 1</TabItem>
+        {null}
+      </Tabs>
+    );
+    const tabs = await screen.findByTestId('tabsTest');
+    expect(tabs.children.length).toEqual(1);
+  });
+
+  it('should log a warning for null children', async () => {
```

Comment:
Should we log a warning for this case? A common conditional pattern I see in react is

```jsx
{someCondition ? <Component /> : null}
```

This would show a warning for this case right?

Reply:
Yea, this would confuse users if they try to do conditional rendering

Reply:
yes that is a good point, I've corrected that so `null` will no longer produce the warning and added another test to ensure that previous cases that were providing that warning are still doing so